### DISCUSSION TITLE
Welcome to uMTP-Responder Discussions!

### DISCUSSION THREAD

**Discussion by jfdelnero** [viveris/uMTP-Responder#50] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Gillou68310** [comment]
@jfdelnero Salut on a discuté aujourd'hui au bureau.
Voici mon implem d'un server mtp pour la nintendo switch!
https://github.com/retronx-team/mtp-server-nx
Ton implem m'a été très utile pour comprendre le protocole merci ;-)

**Comment by kstarfss** [comment]
@jfdelnero Great work on MTP Responder.  I work for a Golf Sports technology company where we use it on our embedded device to transfer data and files between it and PC or game platforms.
